### DISCUSSION TITLE
New translation in Malagasy

### DISCUSSION THREAD

**Discussion by Unviray** [sircharlo/meeting-media-manager#508] [Translations]
I would like to help to translate M³ into a language I speak,Malagasy (MG/mg).

**Comment by mtdvlpr** [comment]
Awesome! @sircharlo can set you up for translating the application. I can add you as a translator for the documentation page. You'll have to sign up for [GitLocalize](https://gitlocalize.com/) using your GitHub account.

**Reply by mtdvlpr** [reply]
Or if you don't want to wait and are familiar with forking, you can fork the repo, create a src/renderer/locales/mg.json file, copy the en.json contents and start translating that way. Then you can create a pull request for it.

**Reply by mtdvlpr** [reply]
P.S. @sircharlo: is it an idea to make me admin in your gitLocalize repo (like I did you for the docs repo)? That way both of us can setup new translators without having to wait :)

**Reply by sircharlo** [reply]
> P.S. @sircharlo: is it an idea to make me admin in your gitLocalize repo (like I did you for the docs repo)? That way both of us can setup new translators without having to wait :)

Of course! I thought you already were; oversight on my part. Added as admin :)

**Comment by sircharlo** [comment]
Thank you so much!

To get started, please [sign into GitLocalize](https://gitlocalize.com/) using your GitHub account, and then let me know when that's done by adding a comment in this chat. 

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Comment by mtdvlpr** [comment]
Hi @Unviray, we encountered a small issue while trying to implement your translations. It seems that one of the packages that we use doesn't support Malagasy. Would you be willing to contribute to their package and add Malagasy to their locale? It's not a lot, just some date/time translations, like the days of the week. You can add an issue at their repo [here](https://github.com/iamkun/dayjs/issues). We need this to display a local date format that includes the full day name.

**Reply by Unviray** [reply]
Sure

**Reply by mtdvlpr** [reply]
Btw, I tested and it is not blocking or breaking. For now it will just take the English day names. So feel free to not translate it. But if you do, let us know so we can enable dayjs for Malagasy.

**Comment by mtdvlpr** [comment]
Hi @Unviray, how's the translating coming along? If you run into any issues, let us know. We're more than willing to help you out!

**Reply by Unviray** [reply]
Sorry, I'm very busy

**Reply by mtdvlpr** [reply]
No worries! Just wanted to make sure you're not stuck and in need of assistance. Take your time! After all, you're mainly helping your own congregation with these translations :). Thanks again for your continuing efforts!
